hw4/02-sessions.js

```diff
@@ -1,15 +1,102 @@
-const express = require('express');
```

Comment:
Your code is super easy to follow and you even made it easier to use than I did because I didn't list links to some possible routes like you did, nice touch! There was only one small issue I saw was that when I typed in `localhost:5000/hello` it gave me a 404 error. 

I actually ended up modeling mine after the 404 route using `app.all()` . If you wanted to do something like that too it might even be super quick to adjust just by changing the guts of the 404 route. 

Literally everything else looks great to me!